apps/flame-defi/app/features/evm-wallet/services/astria-withdrawer-service/astria-withdrawer-service.test.ts

```diff
@@ -123,6 +123,7 @@ describe("GenericContractService", () => {
         functionName: "testMethod",
         args: [],
         value: 100n,
+        account: undefined,
```

Comment:
I think this was added because of unneeded additions to the token approval service that no longer exists, so we can remove it.